scripts/src/markdown.ts

```diff
@@ -1,6 +1,6 @@
 // this is mostly copy-pasted from https://github.com/expo/expo/blob/master/tools/src/Markdown.ts
 
-import { unescape } from 'lodash';
+import _ from 'lodash';
```

Comment:
```suggestion
import unescape from 'lodash/unescape';
```
is this not supported with esm?